robolectric/src/test/java/org/robolectric/shadows/ShadowActivityTest.java

```diff
@@ -1501,6 +1501,22 @@ public void buildActivity_optionBundleWithInvalidNonDefaultDisplaySet_launchesOn
     }
   }
 
+  @Test
+  public void buildActivity_abstractActivityClass_throwsRuntimeException() {
+    Throwable throwable =
+        assertThrows(
+            RuntimeException.class,
+            () -> {
+              ActivityController<AbstractTestActivity> controller =
+                  Robolectric.buildActivity(AbstractTestActivity.class, null);
+              // This line will not be executed.
+              assertThat(controller).isNull();
+            });
+    assertThat(throwable).isInstanceOf(RuntimeException.class);
```

Comment:
`assertThrows` returns a throwable of the type you provide (here `RuntimeException`).
So line 1506 could be `RuntimeException throwable =`, and this check can be removed.